plugins/herald.py

```diff
@@ -45,6 +45,19 @@ def herald(text, nick, chan, db, conn):
         db.commit()
         return("greeting successfully added")
 
+@hook.command()
+def deleteherald(text, chan, db, conn):
+    """deleteherald [nickname] Delete [nickname]'s herald."""
+
+    db_init(db, conn.name)
+    tnick = db.execute("select name from herald where name = :name and chan = :chan", {'name': text.lower(), 'chan': chan.lower()}).fetchone()
+
+    if tnick:
+        db.execute("delete from herald where name = :name and chan = :chan", {'name': text.lower(), 'chan': chan})
+        db.commit()
+        return ("greeting for {} has been removed".format(text.lower()))
+    else:
+        return("{} does not have a herald".format(text.lower()))
```

Comment:
For line 58 and 60 you shouldn't wrap your return in parenthesis `()` as this will return a tuple instead of a string.